pkg/api/v1/types.go

```diff
@@ -1699,4 +1707,5 @@ func ToInternalClusterType(externalClusterType string) kubermaticv1.ClusterType
 
 const (
 	InitialMachineDeploymentRequestAnnotation = "kubermatic.io/initial-machinedeployment-request"
+	UserSSHKeyAgentEnabledAnnotation          = "kubermatic.io/user-ssh-key-agent-enabled"
```

Comment:
What is this used for?

Reply:
Oh this should be removed, I was thinking of using the annotations(temporarily) to avoid `EnableUserSSHKeyAgent` being updated, but since I am gonna use the webhook instead, I will get rid of it